### ISSUE TITLE
Rasbian Install

### ISSUE THREAD

**Issue by swoopsta** [causefx/Organizr#890]
###### Operating System: Windows/MacOS/Ubuntu
<hr>

##### Problem Description:
I can confirm that the Auto Installer works on the latest version of Raspbian on a Raspberry Pi 3.
RASPBIAN STRETCH LITE
Minimal image based on Debian Stretch
Version:April 2018
Release date:2018-04-18

**Comment by elmerfdz** [comment]
Thanks for lettting us know :) I'll update the wiki to reflect that.

**Comment by Blackraz0r** [comment]
cant say the same.

i got this error and the site were not accasible via another machine:

> Nginx vhost template type?:

[CF] CloudFlare
[LE] Let's Encrypt/Standard

- le

> Do you want to generate Let's Encrypt SSL certs as well? [y/n].
- n
chmod: cannot access '/etc/nginx/sites-available/dashboard.nerd-korea.local.conf': No such file or directory
> Site Created for dashboard.nerd-korea.local

/bin/sed: can't read /etc/nginx/sites-available/dashboard.nerd-korea.local.conf: No such file or directory
/bin/sed: can't read /etc/nginx/sites-available/dashboard.nerd-korea.local.conf: No such file or directory
/bin/sed: can't read /etc/nginx/sites-available/dashboard.nerd-korea.local.conf: No such file or directory
[ ok ] Reloading nginx configuration (via systemctl): nginx.service.

######################################################
         Organizr  Install Complete
######################################################

-------------------------------------------------------
           About your Organizr install

**Comment by swoopsta** [comment]
This could be the version of Nginx in the Rasbpian repos as it doesn't sound like the Rasbian-Jessie OS. Try this and see if it works.
`sudo touch /etc/apt/sources.list.d/nginx.list`
Add a reference to the Ubuntu repository, and Debian Jessie backports into nginx.list.
```
deb http://httpredir.debian.org/debian/ jessie-backports main contrib non-free
deb-src http://httpredir.debian.org/debian/ jessie-backports main contrib non-free
deb http://nginx.org/packages/mainline/ubuntu/ xenial nginx 
deb-src http://nginx.org/packages/mainline/ubuntu/ xenial nginx
```
Update and install
```
sudo apt-get update
sudo apt-get install -t jessie-backports openssl
sudo apt-get install nginx
```
If this doesn't work, I'll setup my Pi3 as a dev environment and see if I can help further.

**Comment by elmerfdz** [comment]
@Blackraz0r 's issue should've been fixed in #905 

@causefx you can close this
